viite-UI/src/view/ProjectLinkLayer.js

```diff
@@ -92,6 +92,7 @@
           return !_.isUndefined(selectionTarget.projectLinkData);
```

Comment:
That is related to 373, "User may only apply selection to Segment in progress"

Although later it should allow selecting non-addressed road links as well.